dxp/liferay_component.md

```diff
@@ -44,20 +44,35 @@ Liferay.componentReady([...id]).then(function)
 
 A common use case is the need of run some actions after a component which is not in scope is created.
 
-# `Liferay.destroyComponent()`
+# Lifecycle
 
-This is the safest way to destroy a registered component. It destroys the component registered with the provided component ID. This invokes the component's own destroy lifecycle methods (`destroy` or `dispose`) and deletes the internal references to the component in the component registry.
+We're offering some OOTB lifecycles so you don't need to worry about registering / destroying the components all the time.
 
-```js
-Liferay.destroyComponent(componentId);
-```
+## Registering
 
-# `Liferay.destroyComponents()`
+We automatically register a component in some scenarios:
 
-Destroys registered components matching the provided filter function. If no filter function is provided, it destroys all registered components.
+-   Using `<liferay-frontend:component>` tag. The component will be registered with the provided `componentId`.
+-   Using `<react:component>` tag. The component will be registered with the provided `componentId`.
+-   Rendering a React component with `render` from `frontend-js-react-web`. The component will be registered with the provided `componentId`.
 
-```js
-Liferay.destroyComponents(filterFn);
-```
+This give us the ability to take care of the destroy process as well.
+
+## Destroying
+
+As we said before we provide a mechanism to destroy the registered components: [`Liferay.destroyComponent`](https://github.com/liferay/liferay-portal/blob/0c95d1870808b5fb2ecefe1b32d6fdca97877780/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/component.es.js#L301) and [`Liferay.destroyComponents`](https://github.com/liferay/liferay-portal/blob/0c95d1870808b5fb2ecefe1b32d6fdca97877780/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/component.es.js#L326), which you could benefit fom OOTB just by using what we specified in the Registering point or by properly configuring the component when registering it.
+
+We automatically destroy a component in some scenarios:
+
+-   When a portlet is destroyed, all the components registered with a `portletId` passed on its configuration that matches with the destroyed portlet id are destroyed as well.
```

Comment:
maybe "passed on" → "passed in"

and "its configuration" → "their configuration"

"matches with" → "matches"